src/overlays/gamestates/ovl_file_choose/z_file_choose_NES.c

```diff
@@ -1177,14 +1176,14 @@ u8 sHealthToQuarterHeartCount[] = {
     0, 1, 1, 1, 1, 1, 2, 2, 2, 2, 2, 3, 3, 3, 3, 3,
 };
 s16 sFileSelRupeePrimColors[3][3] = {
-    { 200, 255, 100 },
-    { 170, 170, 255 },
-    { 255, 105, 105 },
+    { 200, 255, 100 }, // Default Wallet
+    { 170, 170, 255 }, // Adult's Wallet
+    { 255, 105, 105 }, // Tycoon´s Wallet
 };
 s16 sFileSelRupeeEnvColors[3][3] = {
-    { 0, 80, 0 },
-    { 10, 10, 80 },
-    { 40, 10, 0 },
+    { 0, 80, 0 },   // Default Wallet
+    { 10, 10, 80 }, // Adult's Wallet
+    { 40, 10, 0 },  // Tycoon´s Wallet
```

Comment:
`Adult Wallet` and `Giant Wallet`

Reply:
I think `Adult's Wallet` and `Giant's Wallet` sounds more natural?

Reply:
This is just what the game calls them, isn't it?
```
[INSTANT:ON]You got an [CL:RED]Adult Wallet[CL:DFLT]![INSTANT:OFF][\n]
[...]
[INSTANT:ON]You got a [CL:RED]Giant Wallet[CL:DFLT]![INSTANT:OFF][\n]
```